src/test/java/com/wikia/webdriver/common/core/api/GraphApi.java

```diff
@@ -0,0 +1,112 @@
+package com.wikia.webdriver.common.core.api;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import org.apache.commons.lang.exception.ExceptionUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.util.EntityUtils;
+import org.openqa.selenium.WebDriverException;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class GraphApi {
+
+    private static String ERROR_MESSAGE = "Problem with Graph API call used to create new facebook test user";
+    private static ArrayList<BasicNameValuePair> PARAMS;
+    private static String wikia_production_app_access_token = "112328095453510|uncl945c48bixtm47AWZM64doDQ";
+
+    public HashMap<String, String> createFacebookTestUser(String appId) {
+        try {
+            HttpResponse response = createTestUser(appId);
+            String entity = EntityUtils.toString(response.getEntity());
+            return new Gson().fromJson(entity, new TypeToken<HashMap<String, String>>(){}.getType());
+        } catch (IOException e) {
+            PageObjectLogging.log("URI_SYNTAX EXCEPTION", ExceptionUtils.getStackTrace(e), false);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "URI_SYNTAX EXCEPTION" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)